### DISCUSSION TITLE
Subscribe the auth.users changes

### DISCUSSION THREAD

**Discussion by Asaraspo** [supabase/supabase#15190] [Questions] [answered]
I am currently implementing the authentication flow. Is it possible to subscribe to the `auth.users` table to automatically detect changes to email_confirmed_at? 

If not I would create a trigger and update a specific field in my `public.users` table and subscribe to that column. Or is there a better solution to this?

Thanks for your help.

**Comment by GaryAustin1** [comment, marked as answer]
You cannot subscribe to auth schema, it is not available to the API at all.  You could certainly use a trigger function to update your public table and use realtime on that.

**Reply by Asaraspo** [reply]
Thanks! Will try and get back.
